src/ConfigurationsProvider.tsx

```diff
@@ -45,16 +45,22 @@ export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) =
   const { data: sueProgress } = useStaticContent({
     refreshTimeKey: 'publicJsonFile-sueReportProgress',
     name: 'sueReportProgress',
-    type: 'json'
+    type: 'json',
+    skip: !Object.keys(sue).length
   });
 
   const mergedConfig = useMemo(() => {
-    const config = {
-      appDesignSystem,
-      sueConfig: { ...sue, ...sueConfigData, sueProgress }
-    };
+    if (!Object.keys(sue).length) {
+      return defaultConfiguration;
+    }
 
-    return mergeDefaultConfiguration({ ...defaultConfiguration }, config);
+    return mergeDefaultConfiguration(
+      { ...defaultConfiguration },
```

Comment:
why spreading the object into a new object? you can use `defaultConfiguration` instead `{ ...defaultConfiguration }`

Reply:
Yes, that's right. I updated the code.

https://github.com/smart-village-solutions/smart-village-app-app/commit/0d2e01bebc02a31441157626adf2da4c2d49d134